templates/sushi_script.erb

```diff
@@ -11,15 +11,18 @@ if [[ -z "$vhost" ]]; then
 	exit 2
 fi
 
+# Remove non-alphanumeric characters
+VHOST_CLEAN=${vhost//[^a-zA-Z0-9-_]/}
```

Comment:
Although the final `-` is taken literally (ie. not within a range) it looks a lot less confusing it it's escaped IMO :)